torch/csrc/distributed/c10d/init.cpp

```diff
@@ -1212,6 +1212,7 @@ that adds a prefix to each key inserted to the store.
       py::call_guard<py::gil_scoped_release>());
 
   module.attr("_DEFAULT_FIRST_BUCKET_BYTES") = ::c10d::kDefaultFirstBucketBytes;
+  module.attr("_DEFAULT_NO_TIMEOUT") = py::cast(kNoTimeout);
```

Comment:
nit: was wondering what's the value add here - this means we need to separately maintain the defaults on cpp and python side anyway (e.g. ppl who changed the signature on cpp side from `kNoTimeout` to `kDefaultTimeout` for example, doesn't automatically updates the python signature. 

if this is the case it might be just easier to put `_DEFAULT_NO_TIMEOUT: timedelta = 0` on the pyi file?

Reply:
I don't think we can do that, since everything defined in the pyi file needs to be explicitly exported from the cpp file. Otherwise there will be a test error.